spring-cloud-data-admin/src/main/resources/META-INF/spring-cloud-data-admin/admin.properties

```diff
@@ -11,3 +11,18 @@ spring.yarn.client.files.0=file:${clouddata.yarn.app.appmaster.path:.}/spring-cl
 spring.yarn.client.files.1=file:${clouddata.yarn.app.container.path:.}/spring-cloud-data-yarn-container-@project.version@.jar
 spring.yarn.client.launchcontext.archiveFile=spring-cloud-data-yarn-appmaster-@project.version@.jar
 spring.yarn.client.resource.memory=1g
+
+# CF Deployer configuration
```

Comment:
: works just fine in properties file. Agreed for consistency though
Le 4 sept. 2015 8:07 PM, "Ilayaperumal Gopinathan" <notifications@github.com>
a écrit :

> In
> spring-cloud-data-admin/src/main/resources/META-INF/spring-cloud-data-admin/admin.properties
> <https://github.com/spring-cloud/spring-cloud-data/pull/60#discussion_r38778463>
> :
>
> > @@ -11,3 +11,18 @@ spring.yarn.client.files.0=file:${clouddata.yarn.app.appmaster.path:.}/spring-cl
> >  spring.yarn.client.files.1=file:${clouddata.yarn.app.container.path:.}/spring-cloud-data-yarn-container-@project.version@.jar
> >  spring.yarn.client.launchcontext.archiveFile=spring-cloud-data-yarn-appmaster-@project.version@.jar
> >  spring.yarn.client.resource.memory=1g
> > +
> > +# CF Deployer configuration
>
> All these properties need to have have = instead of : as the assignment
> operator. With the latest PR, this will change anyway.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spring-cloud/spring-cloud-data/pull/60/files#r38778463>
> .
>